backend/src/main/java/com/google/sps/servlets/RemoveExpiredParticipantsServlet.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.data.Participant;
+import com.google.sps.datastore.ParticipantDatastore;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that removes participant if expired */
+@WebServlet("/api/v1/tasks/remove-expired")
+public class RemoveExpiredParticipantsServlet extends HttpServlet {
+
+  /** Extra padding time in minutes to ensure large enough meeting time block */
+  private static final int PADDING_MINUTES = 10;
+
+  // Get DatastoreService and instiate Match and Participant Datastores
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private final ParticipantDatastore participantDatastore = new ParticipantDatastore(datastore);
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("Remove expired participants");
+
+    List<Participant> unmatchedParticipants = participantDatastore.getUnmatchedParticipants();
+
+    // Check if match exists and not returned yet
+    for (Participant participant : unmatchedParticipants) {
+      if (isExpired(participant)) {
+        participantDatastore.removeParticipant(participant.getUsername());
+        return;
+      }
+    }
+    response.setStatus(HttpServletResponse.SC_OK);
```

Comment:
Continuing from the other PR:

> I think you may be able to delete a batch of entries at once, which would be helpful if we could query datastore for unmatched AND endTimeAvailable. Or else we'd have to loop through the list of unmatchedParticipants and create another list of the ones who are unmatched and expired. I couldn't find detailed documentation about that operation, though. And I was hesitant to try out querying by endTimeAvailable since I had so much trouble doing that last time--I wanted to see if the cron job works before optimizing the query.

I agree that the query is sufficient for now. But at this step, it would still make sense to delete all of the expired participants in one delete operation instead of one transaction per participant. 
https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/DatastoreService.html#delete-java.lang.Iterable-